linux_os/guide/system/network/network-firewalld/ruleset_modifications/configure_firewalld_ports/rule.yml

```diff
@@ -5,33 +5,32 @@ prodtype: ol7,ol8,rhel7,rhel8,rhel9,rhv4,wrlinux1019
 title: 'Configure the Firewalld Ports'
 
 description: |-
-    Configure the <tt>firewalld</tt> ports to allow approved
```

Comment:
Unfortunately, the rule configure_firewalld_port is already modified in PR https://github.com/ComplianceAsCode/content/pull/8497. Notice the discussion at https://github.com/ComplianceAsCode/content/pull/8497#discussion_r843695893 which points to the fact that this rule overlaps with  firewalld_sshd_port_enabled.

Reply:
I would prefer to at the end of the day have both my and your changes, because in my PR I solved the overlap with firewalld_sshd_port_enabled and in your PR there is a good update of the description and rationale. My PR is soon to be merged and after it's merged you can rebase yours and to improve the description and rationale.

Reply:
Great @jan-cerny , lets do it. Thanks